Add missing options to Basic Options (#7090)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

</ConfigKey>

<ConfigKey name="include-source-context" supported={["python"]}>

> You can opt out of source context being included in events sent to Sentry by disabling this option.

Reviewer: ```suggestion
If the source context is being included in events sent to Sentry. The source context are the five lines of code above and below the line of code where an error was happening. 
```

> If this option is enabled, the source context will be included in events sent to Sentry. The source context are the five lines of code above and below the line of code where an error was happening.

Reviewer 2: ```suggestion
When enabled, source context will be included in events sent to Sentry. This source context includes the five lines of code above and below the line of code where an error happened.
```

The option is enabled by default.

</ConfigKey>

<ConfigKey name="include-local-variables" supported={["python"]}>

> If this option is disabled, the SDK will not capture a snapshot of local variables to send with the event.

Reviewer: ```suggestion 
If this option is enabled, the SDK will capture a snapshot of local variables to send with the event.
```

> If this option is enabled, the SDK will capture a snapshot of local variables to send with the event to help with debugging.

Reviewer 2: ```suggestion
When enabled, the SDK will capture a snapshot of local variables to send with the event to help with debugging.
```
